include/flyft/brownian_diffusive_flux.h

```diff
@@ -10,11 +10,16 @@
 namespace flyft
 {
 
-class BrownianDiffusiveFlux : public DiffusiveFlux
+class BrownianDiffusiveFlux : public Flux
     {
     public:
+        TypeMap<double>& getDiffusivities();
+        const TypeMap<double>& getDiffusivities() const;
         void compute(std::shared_ptr<GrandPotential> grand, std::shared_ptr<State> state) override;
         int determineBufferShape(std::shared_ptr<State> state, const std::string& type) override;
+        
+    protected:
+        TypeMap<double> diffusivities_;
```

Comment:
I usually like getters to come after the "main" method. The diffusivities can also be private since we don't anticipate deriving from this class.
```suggestion
        void compute(std::shared_ptr<GrandPotential> grand, std::shared_ptr<State> state) override;

        TypeMap<double>& getDiffusivities();
        const TypeMap<double>& getDiffusivities() const;

        int determineBufferShape(std::shared_ptr<State> state, const std::string& type) override;
        
    private:
        TypeMap<double> diffusivities_;
```